lib/ui/chat/widgets/chat_audio_item.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:audio_waveforms/audio_waveforms.dart';
+import '../../core/themes/colors.dart';
+import '../notifiers/chat_audio_notifier.dart';
+
+class ChatAudioItem extends ConsumerWidget {
+  final String audioPath;
+  const ChatAudioItem({super.key, required this.audioPath});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final state = ref.watch(chatAudioProvider(audioPath));
+    final notifier = ref.read(chatAudioProvider(audioPath).notifier);
+
+    final currentlyPlaying = ref.watch(currentlyPlayingAudioProvider);
+
+    final isThisPlaying = currentlyPlaying == audioPath && state.isPlaying;
+
+    if (!state.isReady) {
+      if (state.error != null) {
+        return SizedBox(
+          height: 50,
+          child: Center(
+            child: Text(
+              state.error!,
+              style: const TextStyle(color: Colors.red, fontSize: 12),
+            ),
+          ),
+        );
+      }
+      return const SizedBox(
+        height: 50,
+        child: Center(child: CircularProgressIndicator(strokeWidth: 2)),
+      );
+    }
+
+    return Row(
+      mainAxisSize: MainAxisSize.min,
+      children: [
+        Container(
+          width: 40,
```

Comment:
I'll suggest we start using .w and .h as appropriate for responsiveness so we don't have to do huge refactors later.